adafruit_rfm9x.py

```diff
@@ -108,118 +93,9 @@ def warn(msg, **kwargs):
-_RH_RF95_PA_RAMP_20US = const(0x0C)
-_RH_RF95_PA_RAMP_15US = const(0x0D)
-_RH_RF95_PA_RAMP_12US = const(0x0E)
-_RH_RF95_PA_RAMP_10US = const(0x0F)
-
-# RH_RF95_REG_0B_OCP                                 0x0b
-_RH_RF95_OCP_ON = const(0x20)
-_RH_RF95_OCP_TRIM = const(0x1F)
-
-# RH_RF95_REG_0C_LNA                                 0x0c
-_RH_RF95_LNA_GAIN = const(0xE0)
-_RH_RF95_LNA_BOOST = const(0x03)
-_RH_RF95_LNA_BOOST_DEFAULT = const(0x00)
-_RH_RF95_LNA_BOOST_150PC = const(0x11)
-
-# RH_RF95_REG_11_IRQ_FLAGS_MASK                      0x11
-_RH_RF95_RX_TIMEOUT_MASK = const(0x80)
-_RH_RF95_RX_DONE_MASK = const(0x40)
-_RH_RF95_PAYLOAD_CRC_ERROR_MASK = const(0x20)
-_RH_RF95_VALID_HEADER_MASK = const(0x10)
-_RH_RF95_TX_DONE_MASK = const(0x08)
-_RH_RF95_CAD_DONE_MASK = const(0x04)
-_RH_RF95_FHSS_CHANGE_CHANNEL_MASK = const(0x02)
-_RH_RF95_CAD_DETECTED_MASK = const(0x01)
-
-# RH_RF95_REG_12_IRQ_FLAGS                           0x12
-_RH_RF95_RX_TIMEOUT = const(0x80)
-_RH_RF95_RX_DONE = const(0x40)
-_RH_RF95_PAYLOAD_CRC_ERROR = const(0x20)
-_RH_RF95_VALID_HEADER = const(0x10)
-_RH_RF95_TX_DONE = const(0x08)
-_RH_RF95_CAD_DONE = const(0x04)
-_RH_RF95_FHSS_CHANGE_CHANNEL = const(0x02)
-_RH_RF95_CAD_DETECTED = const(0x01)
-
-# RH_RF95_REG_18_MODEM_STAT                          0x18
-_RH_RF95_RX_CODING_RATE = const(0xE0)
-_RH_RF95_MODEM_STATUS_CLEAR = const(0x10)
-_RH_RF95_MODEM_STATUS_HEADER_INFO_VALID = const(0x08)
-_RH_RF95_MODEM_STATUS_RX_ONGOING = const(0x04)
-_RH_RF95_MODEM_STATUS_SIGNAL_SYNCHRONIZED = const(0x02)
-_RH_RF95_MODEM_STATUS_SIGNAL_DETECTED = const(0x01)
-
-# RH_RF95_REG_1C_HOP_CHANNEL                         0x1c
-_RH_RF95_PLL_TIMEOUT = const(0x80)
-_RH_RF95_RX_PAYLOAD_CRC_IS_ON = const(0x40)
-_RH_RF95_FHSS_PRESENT_CHANNEL = const(0x3F)
-
-# RH_RF95_REG_1D_MODEM_CONFIG1                       0x1d
-_RH_RF95_BW = const(0xC0)
-_RH_RF95_BW_125KHZ = const(0x00)
-_RH_RF95_BW_250KHZ = const(0x40)
-_RH_RF95_BW_500KHZ = const(0x80)
-_RH_RF95_BW_RESERVED = const(0xC0)
-_RH_RF95_CODING_RATE = const(0x38)
-_RH_RF95_CODING_RATE_4_5 = const(0x00)
-_RH_RF95_CODING_RATE_4_6 = const(0x08)
-_RH_RF95_CODING_RATE_4_7 = const(0x10)
-_RH_RF95_CODING_RATE_4_8 = const(0x18)
-_RH_RF95_IMPLICIT_HEADER_MODE_ON = const(0x04)
-_RH_RF95_RX_PAYLOAD_CRC_ON = const(0x02)
-_RH_RF95_LOW_DATA_RATE_OPTIMIZE = const(0x01)
-
-# RH_RF95_REG_1E_MODEM_CONFIG2                       0x1e
 _RH_RF95_DETECTION_OPTIMIZE = const(0x31)
 _RH_RF95_DETECTION_THRESHOLD = const(0x37)
-_RH_RF95_SPREADING_FACTOR = const(0xF0)
-_RH_RF95_SPREADING_FACTOR_64CPS = const(0x60)
-_RH_RF95_SPREADING_FACTOR_128CPS = const(0x70)
-_RH_RF95_SPREADING_FACTOR_256CPS = const(0x80)
-_RH_RF95_SPREADING_FACTOR_512CPS = const(0x90)
-_RH_RF95_SPREADING_FACTOR_1024CPS = const(0xA0)
-_RH_RF95_SPREADING_FACTOR_2048CPS = const(0xB0)
-_RH_RF95_SPREADING_FACTOR_4096CPS = const(0xC0)
-_RH_RF95_TX_CONTINUOUS_MOE = const(0x08)
-_RH_RF95_AGC_AUTO_ON = const(0x04)
-_RH_RF95_SYM_TIMEOUT_MSB = const(0x03)
```

Comment:
Are these unused const values kept in memory? If they are used, they are usually copied in place and the name dropped. Maybe we need to modify how const works.

Reply:
I'm not sure this made any difference. I took them out since they were not used.